library-no-op/src/main/java/com/chuckerteam/chucker/api/FeatureManager.kt

```diff
@@ -0,0 +1,10 @@
+package com.chuckerteam.chucker.api
+
+import android.content.Context
+
+class FeatureManager constructor(context: Context) {
+    enum class Feature {
+        HTTP_ONLY,
+        HTTP_AND_ERROR
```

Comment:
Instead of HTTP_ONLY and HTTP_AND_ERROR, I would have a more explicit HIDE_ERROR_TAB which would be a boolean in the shared preferences, defaulting to none